File: baekjoon/python/1120.py
x, y = input().split()
x_len = len(x)
sub_len = len(y) - len(x)

diff = x_len
for i in range(sub_len+1):
    temp_diff = 0
    for j in range(x_len):
        if x[j] != y[i+j]: temp_diff += 1
    diff = min(diff, temp_diff)
print(diff)

File: baekjoon/javascript/1120.js
const readline = require("readline");
const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout,
});

let input = [];
rl.on("line", line => {
  input.push(line);
}).on("close", () => {
  const [A, B] = input[0].split(" ");
  const aLen = A.length;
  const subLen = B.length - aLen;

  let diff = aLen;
  for (let i = 0; i < subLen + 1; i++) {
    let tempDiff = 0;
    for (let j = 0; j < aLen; j++) if (A[j] !== B[i + j]) tempDiff++;
    diff = Math.min(diff, tempDiff);
  }
  console.log(diff);
});
